src/main/java/org/xstefank/check/SkipCheck.java

```diff
@@ -25,8 +26,9 @@ private static boolean skipByTitle(JsonNode payload, FormatConfig config) {
 
     private static boolean skipByCommit(JsonNode payload, FormatConfig config) {
         if (config.getFormat().getSkipPatterns().getCommit() != null) {
-            LatestCommitCheck latestCommitCheck = new LatestCommitCheck(config.getFormat().getSkipPatterns().getCommit());
-            return (latestCommitCheck.check(payload) == null);
+            JsonNode commitsJson = GitHubAPI.getJsonWithCommits(payload);
+            Matcher commitMatcher = config.getFormat().getSkipPatterns().getCommit().matcher(commitsJson.get(commitsJson.size() - 1).get(Utils.COMMIT).get(Utils.MESSAGE).asText());
```

Comment:
there is no particular issue with directly calling what I assume LatestCommitCheck is doing. However if commit JSON structure changes (by github, we don't have influence on such things) on have many places will we need to hunt down such change?